MoreLinq.Test/MoreLinq.Test.csproj

```diff
@@ -65,6 +66,7 @@
 
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+    <Content Include="PairsTest.cs" />
```

Comment:
This is not necessary - `.cs` files are included automatically. Please review your Visual Studio settings to understand why it added this line; and remove this line.

Reply:
Thanks. I'll remember this for my next PR (I'm abandoning this one).